### PR TITLE
Create tado-assistant.xml

### PR DESCRIPTION
**Pull request by TerrierX**
Tado Assistant: Your User-Friendly, Free Tado Auto-Assist Alternative

### CODE REVIEW

File: templates/tado-assistant.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <Repository>brainic/tado-assistant:latest</Repository>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/BrainicHQ/tado-assistant/issues</Support>
+  <Project>https://github.com/BrainicHQ/tado-assistant</Project>
+  <Category>HomeAutomation:</Category>
+  <Icon>https://images.ctfassets.net/rro4wewtydcy/2lziJeBTxOCee5n43qBXiz/a03b3387644a3ea67c1a759da2244120/tado_logo_app_icon.webp</Icon>
+  <Config Name="NUM_ACCOUNTS" Target="NUM_ACCOUNTS" Default="1" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="TADO_USERNAME_1" Target="TADO_USERNAME_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="TADO_PASSWORD_1" Target="TADO_PASSWORD_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="true"></Config>
+  <Config Name="CHECKING_INTERVAL_1" Target="CHECKING_INTERVAL_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:13]:
```suggestion
  <Config Name="Account 1 - Checking Interval" Target="CHECKING_INTERVAL_1" Default="15" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">15</Config>
```

File: templates/tado-assistant.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <Config Name="ENABLE_LOG_1" Target="ENABLE_LOG_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:14]:
```suggestion
  <Config Name="Account 1 - Logging" Target="ENABLE_LOG_1" Default="true|false" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: templates/tado-assistant.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Tado-Assistant</Name>
+  <Repository>brainic/tado-assistant:latest</Repository>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/BrainicHQ/tado-assistant/issues</Support>
+  <Project>https://github.com/BrainicHQ/tado-assistant</Project>
+  <Category>HomeAutomation:</Category>
+  <Icon>https://images.ctfassets.net/rro4wewtydcy/2lziJeBTxOCee5n43qBXiz/a03b3387644a3ea67c1a759da2244120/tado_logo_app_icon.webp</Icon>
+  <Config Name="NUM_ACCOUNTS" Target="NUM_ACCOUNTS" Default="1" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:10]:
```suggestion
  <Config Name="Number of Accounts" Target="NUM_ACCOUNTS" Default="1" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">1</Config>
```

File: templates/tado-assistant.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  <Config Name="TADO_USERNAME_1" Target="TADO_USERNAME_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:11]:
```suggestion
  <Config Name="Account 1 - Username" Target="TADO_USERNAME_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: templates/tado-assistant.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  <Config Name="TADO_PASSWORD_1" Target="TADO_PASSWORD_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="true"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:12]:
```suggestion
  <Config Name="Account 1 - Password" Target="TADO_PASSWORD_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="true"/>
```

File: templates/tado-assistant.xml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  <Config Name="CHECKING_INTERVAL_1" Target="CHECKING_INTERVAL_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="ENABLE_LOG_1" Target="ENABLE_LOG_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="MAX_OPEN_WINDOW_DURATION_1" Target="MAX_OPEN_WINDOW_DURATION_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:15]:
```suggestion
  <Config Name="Account 1 - Max Open Window Duration" Target="MAX_OPEN_WINDOW_DURATION_1" Default="" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: templates/tado-assistant.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  <Config Name="TADO_USERNAME_2" Target="TADO_USERNAME_2" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="TADO_PASSWORD_2" Target="TADO_PASSWORD_2" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="true"/>
+  <Config Name="CHECKING_INTERVAL_2" Target="CHECKING_INTERVAL_2" Default="15" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="ENABLE_LOG_2" Target="ENABLE_LOG_2" Default="false" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:19]:
```suggestion
```

File: templates/tado-assistant.xml (hunk 3 above)
Comment (nwithan8) [inline templates/tado-assistant.xml:10]:
```suggestion
  <Icon>https://images.ctfassets.net/rro4wewtydcy/2lziJeBTxOCee5n43qBXiz/a03b3387644a3ea67c1a759da2244120/tado_logo_app_icon.webp</Icon>
  <Requires>
  	Multiple accounts can be added. See documentation for more details: https://github.com/BrainicHQ/tado-assistant?tab=readme-ov-file#-docker-installation
  </Requires>
  <Config Name="NUM_ACCOUNTS" Target="NUM_ACCOUNTS" Default="1" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```

File: templates/tado-assistant.xml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Tado-Assistant</Name>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:3]:
```suggestion
  <Name>Tado Assistant</Name>
```

File: templates/tado-assistant.xml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+  <Repository>brainic/tado-assistant:latest</Repository>
+  <Privileged>false</Privileged>
```
Comment (nwithan8) [inline templates/tado-assistant.xml:5]:
```suggestion
  <Repository>brainic/tado-assistant:latest</Repository>
  <Registry>https://hub.docker.com/r/brainic/tado-assistant</Registry>
  <Privileged>false</Privileged>
```

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/tado-assistant.xml (hunk 6 above)
Comment (TerrierX) [reply templates/tado-assistant.xml]: As per documentation _"MAX_OPEN_WINDOW_DURATION_n: Define the maximum duration (in seconds) for the 'Open Window' detection feature to be active for the nth account. Leave this field empty to use the default duration set in the Tado app."_ I would say it shouldn't be required or?

**Comment by TerrierX** [comment]
@nwithan8 Thanks for all the changes I really appreciate them. I still have a lot to learn.

File: templates/tado-assistant.xml (hunk 6 above)
Comment (nwithan8) [reply templates/tado-assistant.xml]: Sounds good. Please also update the Description with that information so people know what it is.

Comment (TerrierX) [reply templates/tado-assistant.xml]: Done and I have added Descriptions to the other variables as well. Thank you :)

**Pull request merged**